.github/workflows/Helix-PR-Premerge-Check.yml

```diff
@@ -14,13 +14,62 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Verify All Tasks are Finished
-        uses: venkatsarvesh/pr-tasks-completed-action@v1.0.0
+      - name: Label PR if ready to be merged.
+        uses: actions/github-script@v3
         with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Verify the PR has been approved
-        uses: abinoda/label-when-approved-action@1.0.5
-        env:
-          APPROVALS: "1"
-          ADD_LABEL: "CheckedAndApproved"
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const approvalLabel = 'CheckedAndApproved'
```

Comment:
"Label PR if ready to be merged" - there is an existing label "ready to merge", is it what you wanted?

Reply:
No, what I want is to keep the same behavior as the original script.